script/vagrant-provision

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+sudo apt-get update
+
+cd /vagrant/bluetail
+
+#fix dpkg-preconfigure error
+export DEBIAN_FRONTEND=noninteractive
+
+# Install the packages from conf/packages
+xargs sudo apt-get install -qq -y < conf/packages
+xargs sudo apt-get install -qq -y < conf/dev_packages
+# Install some of the other things we need that are just for dev
+sudo apt-get install -qq -y ruby-dev libsqlite3-dev build-essential
+
+# Create postgres database and user.
+sudo -u postgres psql -c "CREATE DATABASE bluetail"
+sudo -u postgres psql -c "CREATE USER bluetail SUPERUSER CREATEDB PASSWORD 'bluetail'"
+
+# Set up Python packages in a virtualenv
+script/bootstrap
+
+# Give permissions to vagrant user on all the packages
+sudo chmod -R ugo+rwx /vagrant/venv
+
+if ! grep -q "SECRET_KEY" ~/.bashrc ; then
+    echo "export SECRET_KEY=$SECRET_KEY" >> /home/vagrant/.bashrc
```

Comment:
What happens if someone defines a `$SECRET_KEY` with spaces in it? Should it be quoted when we assign it?

Reply:
Yeah, could do. There shouldn't be spaces if they know what it's for. Not sure even sure if django will like spaces.
I've got a tidier method so I'll add it in.